test/ui/index.ts

```diff
@@ -3,20 +3,26 @@
 
 import * as os from "os";
 import * as path from "path";
+import * as fse from "fs-extra";
+import * as semver from "semver";
 import { ExTester } from "vscode-extension-tester";
 
 async function main(): Promise<void> {
     try {
         // Run UI command tests
+        const packageContent = await fse.readFile(path.join(__dirname, "..", "..", "..", "package.json"));
+        const packageJSON = JSON.parse(packageContent.toString());
+        let vscodeVersion = packageJSON.engines.vscode || "1.59.0";
```

Comment:
No need to have `"1.59.0"`, if `packageJSON.engines.vscode` is unavailable, that indicates something is wrong in package.json, which is a good signal.